proposals/0035-linalg-matrix.md

```diff
@@ -1196,7 +1226,6 @@ declare void @dx.op.matrixOp(
 Three opcodes are available for this operation class:
 * Matrix Matrix Multiply: `C = A * B`
```

Comment:
Can't leave a comment above since its behind the "expand code" bit but if `matrixOp` is just multiply variants now then `dx.op.matrixMulOp` is worth considering as a name